django/core/management/commands/migrate.py

```diff
@@ -156,6 +161,40 @@ def handle(self, *args, **options):
         else:
             targets = executor.loader.graph.leaf_nodes()
 
+        if options['prune']:
+            if not options['app_label']:
+                raise CommandError('Migrations can be pruned only when an app is specified.')
+            if self.verbosity > 0:
+                self.stdout.write('Pruning migrations:', self.style.MIGRATE_HEADING)
+            to_prune = set(executor.loader.applied_migrations) - set(executor.loader.disk_migrations)
+            squashed_migrations_with_deleted_replaced_migrations = [
+                migration_key for migration_key, migration_obj in executor.loader.replacements.items() if any(
+                    replaced in to_prune for replaced in migration_obj.replaces
+                )
+            ]
+            if squashed_migrations_with_deleted_replaced_migrations:
+                to_prune = []
+                msg = (
+                    '  Pruning cannot take place until the following squashed '
+                    "migrations are recorded as applied (re-run 'manage.py migrate') "
+                    'and have their replaces attribute removed:'
```

Comment:
Quotes should be consistent in a single string:
```suggestion
                    "  Pruning cannot take place until the following squashed "
                    "migrations are recorded as applied (re-run 'manage.py migrate') "
                    "and have their replaces attribute removed:"
```